queries.sql

```diff
@@ -1,11 +1,45 @@
 /*Queries that provide answers to the questions from all projects.*/
+insert into animals(id,name,date_of_birth,escape_attempts,neutered,weigth_kg) values (6,'Plantmon','2021-11-15',2,true,-5.7);
+insert into animals(id,name,date_of_birth,escape_attempts,neutered,weigth_kg) values (7,'Squirtle','1993-03-02',3,false,-12.13);
+insert into animals(id,name,date_of_birth,escape_attempts,neutered,weigth_kg) values (8,'Angemon','2005-06-12',1,true,-45);
+insert into animals(id,name,date_of_birth,escape_attempts,neutered,weigth_kg) values (9,'Boarmon','2005-06-07',7,true,20.4);
+insert into animals(id,name,date_of_birth,escape_attempts,neutered,weigth_kg) values (10,'Blossom','1998-10-13',3,true,17);
+insert into animals(id,name,date_of_birth,escape_attempts,neutered,weigth_kg) values (11,'Ditto','2022-05-14',4,true,22);
 
-SELECT * from animals WHERE name like '%mon';
-select name from animals where date_of_birth between '2016-01-01' and '2019-12-31';
-select name from animals where neutered = true and escape_attempts < 3;
-select date_of_birth from animals where name in ('Agumon','Pikachu');
-select name,escape_attempts from animals where weigth_kg > 10.5;
-select * from animals where neutered = true;
-select * from animals where name not  in ('Gabumon');
-select * from animals where weigth_kg between 10.4 and 17.3;
+begin;
+ALTER TABLE animals RENAME COLUMN species TO unspecified;
```

Comment:
* Kindly verify that the change was made after `species = unspecified` and verify that the species columns went back to the state before the transaction.
 ![image](https://user-images.githubusercontent.com/52497246/207435729-983980bd-510a-4292-af5a-23e563a6981a.png)